pkg/core/config/schema.go

```diff
@@ -24,14 +24,14 @@ const (
 		"$id": "#TenantConfiguration",
 		"type": "object",
 		"properties": {
-			"version": { "const": "2" },
+			"api_version": { "enum": ["v2.0"] },
```

Comment:
Can we use the version string defined in `apiversion`? It seems quite easy to miss when updating the version.